Sources/MobileCoinClient+Async.swift

```diff
@@ -6,7 +6,7 @@ import Foundation
 
 #if swift(>=5.5)
 
-@available(iOS 13.0, *)
+@available(iOSApplicationExtension 15.0.0, *)
```

Comment:
I think this should still be `iOS`

Reply:
Since you're changing these, we might want to do the full specifier like GRPC does:

`@available(macOS 10.15, iOS 13, tvOS 13, watchOS 6, *)`

https://github.com/grpc/grpc-swift/blob/7b6857655d50b4c83cd2191b6135b432106e147c/Sources/GRPC/AsyncAwaitSupport/GRPCClient%2BAsyncAwaitSupport.swift#L20

Reply:
ok ya I think you're correct, this change came from accepting the Xcode suggestion. Ill update it to iOS and add the others